templates/app/code/index.html.twig

```diff
@@ -7,12 +7,10 @@
  #}
 
 {% block content %}
-    {% set route_params = app.request.attributes.get('_route_params')|default({}) %}
-
     <div class="header">
         {% if guest.templates|length == 1 %}
             <h1>{{ 'Get new code'|trans }}</h1>
-            <p class="lead">{{ 'Press %request_access% to get access'|trans({'%request_access%': 'Request access'|trans}) }}</p>
+            <p class="lead">{{ 'Press {request_access} to get access'|trans({request_access: 'Request access'|trans}) }}</p>
```

Comment:
Is the double translations necessary?

```suggestion
            <p class="lead">{{ 'Press Request access to get access'|trans }}</p>
```

Reply:
I translate the placeholder to make sure that the name (“Request access”) in the message exactly matches the actual button text (cf. <https://github.com/itk-dev/dokk1gh/pull/5/files#diff-d831a13f5e1d92ef74bc692564e0cc346f19d067ccb20a96a69fa1ae0ce7de2eR46>). Otherwise, the translator would have to maintain the button text in two places – and we all know what that leads to …